src/node/mod.rs

```diff
@@ -231,6 +231,27 @@ fn devname(dev: dev_t) -> Option<String> {
     Some(String::from_utf8(dev_name).expect("Returned device name is not valid utf8"))
 }
 
+#[cfg(target_os = "openbsd")]
+fn devname(dev: dev_t) -> Option<String> {
+    use std::os::raw::{c_char, c_int};
+    use std::ffi::{CStr};
+
+    let buf: *mut c_char = unsafe {
+        libc::devname(
+            dev,
+            libc::S_IFCHR, // Must be S_IFCHR or S_IFBLK
+        )
+    };
+
+    // Buffer was too small (weird issue with the size of buffer) or the device could not be named.
```

Comment:
That's a stale comment copied from above, since you didn't allocate a "too small" buffer yourself :)

Reply:
Indeed